TuneUp/ProfiledNodeViewModel.cs

```diff
@@ -62,23 +60,51 @@ internal set
         /// <summary>
         /// Indicates whether this node represents the total execution time for its group
         /// </summary>
-        public bool IsGroupExecutionTime
+        public bool IsGroupExecutionTime => NodeModel == null && GroupModel == null;
+
+        /// <summary>
+        /// Getting the original name before graph author renamed the node
+        /// </summary>
+        private static string GetOriginalName(NodeModel node)
```

Comment:
build is failing with the error: Type 'ProfiledNodeViewModel' already defines a member called 'GetOriginalName' with the same parameter types

Reply:
good to go now